src/models/core/AnswersConfig.ts

```diff
@@ -44,7 +44,7 @@ export interface BaseAnswersConfig {
  * @public
  */
 export interface AnswersConfigWithApiKey extends BaseAnswersConfig {
-  /** The api key of the answers experience. */
+  /** The api key of the answers experience which will be sent as a query param */
```

Comment:
small nit: In core we always put a period at the end of the sentences because it works better with the generated docs